src/fxcorrelator.py

```diff
@@ -193,6 +193,130 @@ def initialise(self, program=True, tvg=False, fake_digitiser=False):
         # set an initialised flag
         self._initialised = True
 
+    def _sensor_cb_flru(self, sensor):
+        """
+        Sensor call back function for f-engine LRU
+        :param sensor:
+        :return:
+        """
+        host_name = sensor.name.split('_')[2]
+        result = self.fhosts[host_name].host_okay()
+        sensor.set(time.time(), Sensor.NOMINAL if result else Sensor.ERROR, result)
+        self.logger.info('_sensor_cb_flru ran on %s' % (host_name))
+        Timer(10, self._sensor_cb_flru, sensor).start()
```

Comment:
I see you are starting a new thread for each sensor update. This is quite inefficient since threads are relatively expensive (at an OS level) to create. So we could perhaps use tornado here as discussed in person.